src/core/tsi/alts/crypt/aes_gcm.cc

```diff
@@ -196,7 +201,7 @@ static grpc_status_code aes_gcm_derive_aead_key(uint8_t* dst,
     return GRPC_STATUS_INTERNAL;
   }
   HMAC_CTX_cleanup(&hmac);
-#else
+#elif OPENSSL_VERSION_NUMBER < 0x30000000L
```

Comment:
Should this be `#if` rather than `#elif`?

Reply:
It's part of an #if/#elif/#else that's structured
```
#if OpenSSL < 1.1
// 1.02 impl
#elif OpenSSL < 3
// 1.1 impl
#else
// 3 impl
#endif
```

Reply:
Ah sorry I didn't expand the file enough.